lib/logger/utils.ts

```diff
@@ -90,38 +91,54 @@ export default function prepareError(err: Error): Record<string, unknown> {
   return response;
 }
 
-export function sanitizeValue(_value: unknown, seen = new WeakMap()): any {
-  let value = _value;
-  if (Array.isArray(value)) {
-    const length = value.length;
-    const arrayResult = Array(length);
-    seen.set(value, arrayResult);
-    for (let idx = 0; idx < length; idx += 1) {
-      const val = value[idx];
-      arrayResult[idx] = seen.has(val)
-        ? seen.get(val)
-        : sanitizeValue(val, seen);
-    }
-    return arrayResult;
+type NestedValue = unknown[] | object;
```

Comment:
It should be safer to use `object` instead of `Record<string, unknown>`, since we want sanitize class instances too.